ext/semian/semian.c

```diff
@@ -167,13 +134,11 @@ static VALUE
 update_ticket_count(update_ticket_count_t *tc)
 {
   short delta;
-  struct timespec ts = { 0 };
-  ts.tv_sec = kInternalTimeout;
 
   if (get_max_tickets(tc->sem_id) != tc->tickets) {
     delta = tc->tickets - get_max_tickets(tc->sem_id);
 
-    if (perform_semop(tc->sem_id, kIndexTickets, delta, 0, &ts) == -1) {
+    if (perform_semop(tc->sem_id, kIndexTickets, delta, 0) == -1) {
```

Comment:
@csfrancis Good catch. Would it be acceptable if we changed this call to not pass `IPC_NOWAIT` (so it'll block until there are enough tickets to lower the max count)?